lib/model/message.dart

```diff
@@ -143,7 +154,9 @@ class MessageStoreImpl with MessageStore {
     // The interaction between the fields of these events are a bit tricky.
     // For reference, see: https://zulip.com/api/get-events#update_message
 
-    if (event.origTopic == null) {
+    bool isTopicUpdated = event.origTopic != null;
+
+    if (!isTopicUpdated) {
       // There was no move.
```

Comment:
This seems less clear to me.

In fact I think this name is actively misleading: as the comment says, if `origTopic` is null then there was no move — not a change of topic, and not a change of channel either. That's not obvious (which is why there's a comment about it), but it's a fact about the API as documented at the link in the comment above.